src/main/java/org/mitre/caasd/commons/fileutil/FileUtils.java

```diff
@@ -50,9 +50,6 @@
 
 public class FileUtils {
 
-    private FileUtils() {
```

Comment:
This is a static utility class - standard practice is the keep the constructor and have it throw `IllegalStateException` (at least according to most linters).

Reply:
I've flip flopped on this.  My current view is it's slightly better to keep the class file cleaner.  I no longer think preventing someone from calling ` new FileUtils()` is a relevant critique.  That said.  I do not care much at all.  seems like argueing about the least important thing ever

Reply:
Yeah I don't really care either - just noting the convention or whatever you want to call it.